assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -447,6 +473,22 @@ export default Component.extend({
     }
   },
 
+  scrollToSeparator(indicatorClass) {
+    const separatorEl = this._scrollerEl.querySelector(
+      `.chat-message-separator.${indicatorClass}`
+    );
+    const aboveIndicatorOffset = this.element.clientHeight / 7;
+
+    if (separatorEl) {
```

Comment:
This is weird to me because you access the DOM outside of after render block to check if you should do the operation. All of this should be in afterRender block

Reply:
I fixed it on the latest commit.